installer/data/template/.origintrail_noderc_one_click_mainnet.json

```diff
@@ -57,7 +76,7 @@
               },
               "publicCurrent":{
                 "url":"http://localhost:9999",
-                "name":"public-current",
+                "name":"kb",
```

Comment:
@Mihajlo-Pavlovic  double check this change

Reply:
@Niks988  we moved to using public-current here it should stay that way